docs/source/working_with_aiida/troubleshooting.rst

```diff
@@ -33,14 +36,19 @@ the source of the problem.
   The list of the levels and their order is the same of the `standard python
   logging module <https://docs.python.org/2/library/logging.html#logging-levels>`_.
```

Comment:
We should add here that in between the standard `INFO` and `WARNING` we have our custom `REPORT` level which is also the default.

Reply:
done